src/MaterialExtension.cpp

```diff
@@ -1,14 +1,17 @@
 #include "MaterialExtension.h"
+#include "MaterialParameters.h"
 
+using namespace MaterialParameters;
 
 MaterialExtension::MaterialExtension(const G4String& name, const G4Material* baseMaterial)
 : G4Material(name,baseMaterial->GetDensity(),baseMaterial,    
    baseMaterial->GetState(),baseMaterial->GetTemperature(),
    baseMaterial->GetPressure())
 {
-    foPsPobability = 0.;
     foPslifetime = 0.;
+    foPsFraction = 0.;
```

Comment:
add description or change name of variables - it is not clear what "fo" means.

Reply:
oPs like ortho-positronium